src/components/onboarding/OnboardingFlow.tsx

```
@@ -52,8 +60,495 @@ interface OnboardingFlowProps {
+                                <XCircle className="h-5 w-5 text-red-500" />
+                            </Button>
+                        </div>
+                    ))}
+                </div>
+            </StepCard>
+        );
+
+      case 4: // Old Home
+      case 5: // New Home
+        const homeKey = currentStep === 4 ? 'oldHome' : 'newHome';
+        const homeData = data[homeKey] as HomeDetails;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)